mlflow/langchain/utils/chat.py

```diff
@@ -177,10 +176,7 @@ def try_transform_response_to_chat_format(response: Any) -> dict[str, Any]:
                 total_tokens=None,
             ),
         )
-        if IS_PYDANTIC_V2_OR_NEWER:
-            return transformed_response.model_dump(mode="json", exclude_unset=True)
-        else:
-            return json.loads(transformed_response.json(exclude_unset=True))
+        return transformed_response.model_dump(mode="json", exclude_unset=True)
     else:
```

Comment:
ℹ️ Simplified conditional: removed the `if IS_PYDANTIC_V2_OR_NEWER: ... else: ...` pattern. Now always uses `.model_dump()` instead of conditionally using `.json()` for v1.